scripts/release/monitor_release/settings.py

```diff
@@ -0,0 +1,233 @@
+import os
+from pathlib import Path
+from typing import Final
+
+from dotenv import load_dotenv
+from pydantic import BaseModel, Field, HttpUrl, TypeAdapter, model_validator
+from pydantic_settings import BaseSettings, SettingsConfigDict
+
+from .models import Deployment
+
+# NOTE: this mapping suggests that this script might be rather living a in a
+# different repo
+#
+_DEPLOYMENTS_MAP = {
+    Deployment.master: "osparc-master.speag.com",
```

Comment:
we should stop calling the deployments by their "trivial names" (this is chemistry lab jargon), and always use the FQDN (so e.g. osparc-master.speag.com, the primary domain) as the name. At least in code. Not sure tif this is the time and place to do it.

Reply:
Yes. This is a tmp map between the "old" and the osparc-config names.
I just have this here to check against "legacy" code. But once it is away, it will be deleted